eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -74,22 +70,11 @@ jobs:
           pip install doc-warden==$(DocWardenVersion)
           ward scan -d $(Build.SourcesDirectory) -c $(Build.SourcesDirectory)/eng/.docsettings.yml
         displayName: "Verify Readmes"
-      - task: UseDotNet@2
-        displayName: "Use .NET Core sdk $(DotNetCoreSDKVersion)"
-        inputs:
-          version: "$(DotNetCoreSDKVersion)"
-      - task: UseDotNet@2
+      - task: DotNetCoreInstaller@2
         displayName: "Use .NET Core runtime $(DotNetCoreRuntimeVersion)"
         inputs:
           packageType: runtime
           version: "$(DotNetCoreRuntimeVersion)"
-      - task: DownloadPipelineArtifact@2
```

Comment:
Removed this since its not being used. It seems to have been added for running the dependency report but that has been moved to the aggregate report pipeline.